### DISCUSSION TITLE
Bloodstained curse of the moon 2 won't open

### DISCUSSION THREAD

**Discussion by Tuliox99** [skyline-emu/skyline#1181] [Q&A]
Bloodstained curse of the moon 2 is a very good 8-bit game of the Metroidvania genre. I downloaded the rom from ziperto.com and when I open it in the Skyline emulator it doesn't start the game, is there a solution? The game is very fun I appreciate the help you can provide me.

**Comment by s3lverd** [comment]
No piracy, you should get the games from your own switch for support
